nni/compression/pytorch/utils/counter.py

```diff
@@ -313,12 +313,13 @@ def format_results(self):
 
         table.field_names = headers
         for i, result in enumerate(self.results):
+            flops_count = int(result['flops'].item()) if isinstance(result['flops'], torch.Tensor) else int(result['flops'])
```

Comment:
`torch.is_tensor(...)`

Reply:
It seems that line 366 in the same file also uses isinstance for checking. Also, do we need to consider the recommendation from the official website? [https://pytorch.org/docs/stable/generated/torch.is_tensor.html](url)

Reply:
In the official recommendation is isinstance, I'm okay with that.